telegram-bot-bumblebee-core/src/main/resources/application.yml

```diff
@@ -31,6 +32,8 @@ bumblebee:
 # Commands
 ## Youtube search
 youtube:
+  webhook:
+    url:
```

Comment:
Seems to be redundant. In case we really do need it then we should perform some refactoring: host url should be defined on config root level only once. Code will use this value and add specific paths to produce absolute url to service.